core/lib/contract_verifier/src/tests/real.rs

```diff
@@ -224,6 +224,53 @@ async fn using_standalone_solc(specify_contract_file: bool) {
     assert_eq!(output.abi, counter_contract_abi());
 }
 
+#[test_casing(3, [(Some(100), None), (None, Some("shanghai")), (Some(200), Some("paris"))])]
+#[tokio::test]
+async fn using_standalone_solc_with_custom_settings(
+    (runs, evm_version): (Option<u16>, Option<&str>),
```

Comment:
Nit: `test_casing` supports multiple args, so it's possible to define `runs` and `evm_version` as separate args here.

Reply:
Nice!